style.css

```diff
@@ -50,11 +50,13 @@ em.word:before {
 nav.links {
   display: flex;
   flex-direction: column;
+  width: 100vw;
```

Comment:
I think it is a bit contradictory for us to set `width: 100vw;` on the `.links` element and then make all its children have `padding-left: 8px;` (line 59).

It could be simpler if the padding was on the parent element.

Also, the `body` itself already have a padding, so this property might be making the `.links` be wider than the space it should fill…

Reply:
Just moved the `padding` from child elements to the parent one. :smiley: